modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogManager.java

```diff
@@ -29,16 +29,16 @@ public interface CatalogManager extends IgniteComponent, CatalogService {
      * Executes given command.
      *
      * @param command Command to execute.
-     * @return Future representing result of execution.
+     * @return Future representing result of execution (it will be completed with the created catalog version).
      */
-    CompletableFuture<Void> execute(CatalogCommand command);
+    CompletableFuture<Integer> execute(CatalogCommand command);
```

Comment:
Didn’t notice where you are using the result of the future, not in the test code?

Reply:
In `DdlCommandHandler#handleCreateIndex()`